src/app/pages/study/study.component.html

```diff
@@ -0,0 +1,68 @@
+    <app-toolbar>
+      <div start>
+        <button [routerLink]="['/']" md-icon-button>
+          <i class="material-icons">arrow_back</i>
+        </button>
+        <div class="title">Study {{studyId}}</div>
+      </div>
+      <div end>
+        <button md-icon-button><i class="material-icons">more_vert</i></button>
+      </div>
+    </app-toolbar>
+    <div class="content">
+    <p *ngIf="!(isLoaded$ | async)">Loading...</p>
+      <ng-container *ngIf="isStudyLoadedAndValid$ && isLoaded$ | async">
+        <!--<p *ngFor="let subject of (subjects$ | async)">
```

Comment:
avoid leaving commented code behind, remove it completely. if we need it back for some reason we use the git's history.